packages/client-sdk-nodejs/src/internal/leaderboard-client.ts

```diff
@@ -60,11 +61,12 @@ export class LeaderboardClient implements InternalLeaderboardClient {
   public async leaderboardUpsert(
     cacheName: string,
     leaderboardName: string,
-    elements: Map<bigint, number>
+    elements: Map<bigint | number, number>
```

Comment:
thanks for doing this! two things:

1. I just realized - missed in my first review - that you are using Map here, and typically in JS it's more common/ergonomic for people to use `Record`. Ultimately, ideally, we probably want to support both. But for now, to start, we should just pick one and I think the one we probably want to pick by default is `Record`.
2. I'm assuming you tested whether the compiler would accept a `number` input for a `bigint` argument, and that that resulted in a compilation failure?

Reply:
1. Record doesn't accept bigint as a key type, so it would have to be `Record<string|number, number>`, is that acceptable? Could switch the Map version to be the same type as well.
2. Yeah, got the error: `Type 'number' is not assignable to type 'bigint'`

Reply:
re: 1, ah that makes sense. in that case let's keep what you've got here for now.

re: 2, thanks for confirming!